### DISCUSSION TITLE
Difference between CYCLE_IMAGE_TEXT_LIST and CYCLE_LIST

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#171] [Developer Feedback]
There are two controls:

- [CYCLE_LIST](https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/CYCLE_LIST/)
- [CYCLE_IMAGE_TEXT_LIST](https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/CYCLE_IMAGE_TEXT_LIST/)

Second one is able to display images. From documentation it's not clear what are use cases for each control.

**Comment by MyoungXUE** [comment]
Thank you for your feedback. I will confirm the Widget functionality and update the documentation as soon as possible.

**Comment by MyoungXUE** [comment]
Hi, I have updated the documentation.

https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/CYCLE_IMAGE_TEXT_LIST/

`CYCLE_IMAGE_TEXT_LIST` can put one image and text per line

`CYCLE_LIST` can only put images

Also updated the showcase sample to run in the simulator for a more visual understanding of the differences between widgets https://docs.zepp.com/docs/samples/app/showcase/
